### DISCUSSION TITLE
Where to configure passwordFromEnv

### DISCUSSION THREAD

**Discussion by Savitha-Amaldev** [kedacore/keda#1857] [Q&A / Need Help] [answered]
I'm pretty new to keda. I'm writing a scaledobject with MySQL trigger and I saw "passwordFromEnv" where password needs to be mentioned. Can someone tell me where are we configuring this environment variable specific to keda?

I have a secret.yaml where i configured the password in base64 encoded form and deployed the same.  But scaledobject READY and ACTIVE status is still unknown. 

Is there any way to check whether my scaledobject is able to establish a connection with MSSQL trigger?

**Comment by tomkerkhove** [comment, marked as answer]
Duplicate of https://github.com/kedacore/keda-docs/discussions/461
